### ISSUE TITLE
Add helper on callback to make it work faster

### ISSUE THREAD

**Issue by Coletz** [ztc1997/Fingerprint2Sleep#14]
Hi, I'd suggest you to add the override to onAuthenticationHelp inside the authenticationCallback. It can handle fast touch to the fp (code 5 == finger moved too fast)

```
override fun onAuthenticationHelp(code: Int, help: CharSequence){
    // Check if enrolled_only option is enabled, if not call the function to trigger fp action
}
```

Thanks for your hard work man!

**Comment by ztc1997** [comment]
Thank you for your suggestion, but onAuthenticationHelp has been overrided in version 0.5.0. After my test, different devices will report different errorCode when swiping fast, and many devices will not report code 5. So all calls to this method are now considered to be fast swipe.

**Comment by Coletz** [comment]
Android fragmentation strikes again :\
0.5.0 code isn't available? I see only 0.1.0, 0.3.0 and 0.3.1 released, but I can't understand if the code is updated or if you've stop updating after 0.3.1

**Comment by ztc1997** [comment]
Already decided to open source and upload the latest source code, welcome made more suggestions.
